src/Hedgehog/paket.template

```diff
@@ -1,26 +0,0 @@
-type
-    project
-id
-    Hedgehog
-version
-	0.8.3.0
-description
-    Hedgehog will eat all your bugs.
-authors
-    Jacob Stanley, Nikos Baxevanis
-copyright
-    Copyright 2017
-summary
-    An alternative property-based testing system for F#, in the spirit of QuickCheck. The key improvement is that shrinking comes for free.
-licenseurl
-    https://raw.githubusercontent.com/hedgehogqa/fsharp-hedgehog/master/LICENSE
-projecturl
-    https://github.com/hedgehogqa/fsharp-hedgehog
-iconUrl
-    https://github.com/hedgehogqa/fsharp-hedgehog/raw/master/img/hedgehog-logo.png
-tags
-    fsharp, f#, testing
-files
```

Comment:
@ThisFunctionalTom any ideas if this is possible with `dotnet pack`?

Reply:
> I just converted paket.template to NuGet metadata properties and pushed to your branch. (There's still the [`files`](https://github.com/hedgehogqa/fsharp-hedgehog/pull/207#pullrequestreview-445389964) field missing but that isn't yet part of any release, so it can be migrated also separately.)

Did you answer your own question here?...or is there still something I could look into?

Reply:
I meant that we've taken care of everything except the `files` field which basically copies the sources into the NuGet Package so they can be consumed by Fable.

This was originally added by @ThisFunctionalTom in https://github.com/hedgehogqa/fsharp-hedgehog/pull/194/files#diff-5cc85c81aa919cd092437e0ef6744e25 and I think the equivalent in `dotnet pack` is the [`IncludeSource`](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#includesource) property.

It's the only thing left to do, and I'm planning to look into it soon (perhaps even today). After that we should be good.